pallets/offchain-worker/src/lib.rs

```diff
@@ -146,8 +151,9 @@ decl_module! {
 		const QuerySessionLength: u32 = T::QuerySessionLength::get();
 		const OcwQueryReward: BalanceOf<T> = T::OcwQueryReward::get();
 
+		// benchmark is 16 us in personal mac 
```

Comment:
Any plan to make it again in other pcs?
Then fire an issue

Reply:
will do it with hardware available before the testnet.